ccloader/js/mod.js

```diff
@@ -5,9 +5,9 @@ export class Mod {
 	 *
 	 * @param {import('./ccloader').ModLoader} modloader
 	 */
-	constructor(modloader){
+	constructor(modloader, modset = ''){
```

Comment:
I get that `modset` is good for `enableKey` but since this value is visible to mods I would prefer it defaulting to `default` with some extra handling in `enableKey`.